### PR TITLE
Hindi translation of webgl contribution guide

### PR DESCRIPTION
**Pull request by ranaaditya**
Resolves Hindi Translation of WebGL Contribution Guide

 Changes: **Add webgl_contribution_guide.md in contributor_docs/hi/ folder**

 Screenshots of the change:

#### PR Checklist

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated
- [x] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Comment by ranaaditya** [comment]
@TanviKumar @Divyansh013 can you please review this PR and let me know the changes if any.

Thanks !

**Review by Divyansh013 — CHANGES_REQUESTED** [review]
Hey @ranaaditya have left some reviews please look into that. Thanks for your PR :)

File: contributor_docs/hi/webgl_contribution_guide.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+# वेबजीएल योगदान गाइड
+
+नमस्ते, यदि आप इस पृष्ठ को पढ़ रहे हैं, तो आप शायद वेबजीएल मोड पर काम करने में मदद करने के इच्छुक हैं। आपकी मदद के लिए धन्यवाद, हम आपकी सहायता के लिए आभारी हैं! इस पृष्ठ का उद्देश्य वेबजीएल योगदानों को संरचित करने का तरीका समझाना और परिवर्तन करते समय कुछ सुझाव प्रदान करना है।
+
+## संसाधन
+
+- हमारे [वेबजीएल मोड की संरचना अवलोकन](https://github.com/processing/p5.js/blob/main/contributor_docs/webgl_mode_architecture.md) पढ़ें ताकि समझा जा सके कि 2डी मोड से वेबजीएल मोड कैसे भिन्न है। इससे शेडर, स्ट्रोक और अन्य के लिए कुछ कार्यान्वयन विशेषताओं को समझने में मदद मिलेगी।
+- हमारे [योगदानकर्ता दिशानिर्देश](https://p5js.org/contributor-docs/#/./contributor_guidelines) पढ़ें जिससे यह पता चलेगा कि मुद्दों को कैसे बनाया जाए, कोडबेस को कैसे सेट किया जाए और परिवर्तनों को कैसे परीक्षित किया जाए।
```
[the remark this answers is not in the document]
Comment (ranaaditya) [reply contributor_docs/hi/webgl_contribution_guide.md:8]: resolved :+1:

File: contributor_docs/hi/webgl_contribution_guide.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+- यह जानना भी उपयोगी हो सकता है कि ब्राउज़र के वेबजीएल एपीआई के बारे में थोड़ा जानना, जिस पर पी5.जेएस का वेबजीएल मोड बनाया गया है:
```
[the remark this answers is not in the document]
Comment (ranaaditya) [reply contributor_docs/hi/webgl_contribution_guide.md:9]: resolved :+1:

**Comment by ranaaditya** [comment]
Hi @Divyansh013 I have fixed the suggested changes, can you review and let me know if something needs to be fixed in it.

Thanks,
Aditya

**Review by Divyansh013 — APPROVED** [review]

**Comment by Qianqianye** [comment]
Thanks @ranaaditya for translating and @Divyansh013 for reviewing. I will merge it after @TanviKumar approves the translation since all the translations need to be approved by 2 translation stewards. Thanks!

**Review by TanviKumar — APPROVED** [review]

**Comment by TanviKumar** [comment]
@Qianqianye this PR can be merged

**Comment by Qianqianye** [comment]
Looks great, thanks @ranaaditya @Divyansh013 @TanviKumar  💗
@all-contributors please add @ranaaditya for translation

**Pull request merged**